packages/abi-gen/test-cli/fixtures/contracts/AbiGenDummy.sol

```diff
@@ -135,7 +137,28 @@ contract AbiGenDummy
     function nestedStructInput(NestedStruct memory n) public pure {}
     function nestedStructOutput() public pure returns(NestedStruct memory) {}
 
+    struct StructNotDirectlyUsedAnywhere {
```

Comment:
suggestion:

```
StructNotInMethodSignatures
StructNotInMethodSignaturesNested | NestedStructNotInMethodSignatures
StructNotInMethodSignaturesNestedOutput
```

I'm not attached to this naming convention, but would like to establish some kind of convention.

Maybe we could use `output/input` for all the method names that test return types? I.e. `<ReturnType>Output` and `<ReturnType>Input`

Reply:
Can we punt on establishing some conventions in this contract?  I agree that we definitely should have some more organization here, but it feels out of scope for this PR, especially given the looming milestone deadline.  Feel free to raise a `tech-debt` issue for it :)